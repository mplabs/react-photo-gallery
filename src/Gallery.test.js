import React from 'react'
import { render, screen } from '@testing-library/react'

import { photos } from '../__test__/test-photo-data'
import Gallery from './Gallery'

const handleClick = jest.fn()
const columns = jest.fn((_) => 3)

test('Gallery renders and reacts to ckick events', async () => {
  render(<Gallery photos={photos} onClick={handleClick} />)

  const image = screen.queryAllByRole('img')[0]
  image.click()
  expect(handleClick).toHaveBeenCalled()
})

test('Gallery renders correctly with direction set to column', () => {
  const wrapper = render(<Gallery photos={photos} direction="column" />)
  expect(wrapper).toMatchSnapshot()
})

test('Gallery renders correctly with a column function', () => {
  const wrapper = render(
    <Gallery photos={photos} columns={columns} direction="column" />
  )
  expect(columns).toBeCalledWith(1140)
})
