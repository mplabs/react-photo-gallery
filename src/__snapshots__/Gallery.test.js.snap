// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gallery renders correctly with direction set to column 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="react-photo-gallery--gallery"
      >
        <div
          style="position: relative; height: 2007.1px;"
        >
          <img
            alt="DSC02576"
            height="250.4"
            sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
            src="https://live.staticflickr.com/65535/40673436833_5fc1ab52fe_b.jpg"
            srcset="https://live.staticflickr.com/65535/40673436833_5fc1ab52fe.jpg 500w,https://live.staticflickr.com/65535/40673436833_5fc1ab52fe_c.jpg 800w,https://live.staticflickr.com/65535/40673436833_5fc1ab52fe_b.jpg 1024w,https://live.staticflickr.com/65535/40673436833_6f245db6aa_h.jpg 1600w"
            style="margin: 2px; display: block; position: absolute; left: 0px; top: 0px;"
            title="DSC02576"
            width="375.7"
          />
          <img
            alt="DSC02565"
            height="250.4"
            sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
            src="https://live.staticflickr.com/65535/47587238132_4683fdabb2_b.jpg"
            srcset="https://live.staticflickr.com/65535/47587238132_4683fdabb2.jpg 500w,https://live.staticflickr.com/65535/47587238132_4683fdabb2_c.jpg 800w,https://live.staticflickr.com/65535/47587238132_4683fdabb2_b.jpg 1024w,https://live.staticflickr.com/65535/47587238132_54f98ac5e9_h.jpg 1600w"
            style="margin: 2px; display: block; position: absolute; left: 379.7px; top: 0px;"
            title="DSC02565"
            width="375.7"
          />
          <img
            alt="DSC02548"
            height="250.4"
            sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
            src="https://live.staticflickr.com/65535/32697497947_7d6dfaf5ec_b.jpg"
            srcset="https://live.staticflickr.com/65535/32697497947_7d6dfaf5ec.jpg 500w,https://live.staticflickr.com/65535/32697497947_7d6dfaf5ec_c.jpg 800w,https://live.staticflickr.com/65535/32697497947_7d6dfaf5ec_b.jpg 1024w,https://live.staticflickr.com/65535/32697497947_81c6d9139e_h.jpg 1600w"
            style="margin: 2px; display: block; position: absolute; left: 759.3px; top: 0px;"
            title="DSC02548"
            width="375.7"
          />
          <img
            alt="DSC02540"
            height="250.4"
            sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
            src="https://live.staticflickr.com/65535/32697497647_31cd88c0b5_b.jpg"
            srcset="https://live.staticflickr.com/65535/32697497647_31cd88c0b5.jpg 500w,https://live.staticflickr.com/65535/32697497647_31cd88c0b5_c.jpg 800w,https://live.staticflickr.com/65535/32697497647_31cd88c0b5_b.jpg 1024w,https://live.staticflickr.com/65535/32697497647_7a36437b05_h.jpg 1600w"
            style="margin: 2px; display: block; position: absolute; left: 0px; top: 254.4px;"
            title="DSC02540"
            width="375.7"
          />
          <img
            alt="DSC02535"
            height="250.4"
            sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
            src="https://live.staticflickr.com/65535/47587238242_6776b8a582_b.jpg"
            srcset="https://live.staticflickr.com/65535/47587238242_6776b8a582.jpg 500w,https://live.staticflickr.com/65535/47587238242_6776b8a582_c.jpg 800w,https://live.staticflickr.com/65535/47587238242_6776b8a582_b.jpg 1024w,https://live.staticflickr.com/65535/47587238242_feab7e0aa0_h.jpg 1600w"
            style="margin: 2px; display: block; position: absolute; left: 379.7px; top: 254.4px;"
            title="DSC02535"
            width="375.7"
          />
          <img
            alt="DSC02520"
            height="250.4"
            sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
            src="https://live.staticflickr.com/65535/40673437163_703ffe416d_b.jpg"
            srcset="https://live.staticflickr.com/65535/40673437163_703ffe416d.jpg 500w,https://live.staticflickr.com/65535/40673437163_703ffe416d_c.jpg 800w,https://live.staticflickr.com/65535/40673437163_703ffe416d_b.jpg 1024w,https://live.staticflickr.com/65535/40673437163_7f4a4d66f2_h.jpg 1600w"
            style="margin: 2px; display: block; position: absolute; left: 759.3px; top: 254.4px;"
            title="DSC02520"
            width="375.7"
          />
          <img
            alt="DSC02494"
            height="250.4"
            sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
            src="https://live.staticflickr.com/65535/47587238362_5d355937cb_b.jpg"
            srcset="https://live.staticflickr.com/65535/47587238362_5d355937cb.jpg 500w,https://live.staticflickr.com/65535/47587238362_5d355937cb_c.jpg 800w,https://live.staticflickr.com/65535/47587238362_5d355937cb_b.jpg 1024w,https://live.staticflickr.com/65535/47587238362_65fe216f4b_h.jpg 1600w"
            style="margin: 2px; display: block; position: absolute; left: 0px; top: 508.8px;"
            title="DSC02494"
            width="375.7"
          />
          <img
            alt="DSC02492"
            height="197.7"
            sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
            src="https://live.staticflickr.com/65535/32697498187_7a74849c76_b.jpg"
            srcset="https://live.staticflickr.com/65535/32697498187_7a74849c76.jpg 500w,https://live.staticflickr.com/65535/32697498187_7a74849c76_c.jpg 800w,https://live.staticflickr.com/65535/32697498187_7a74849c76_b.jpg 1024w,https://live.staticflickr.com/65535/32697498187_9d07b79e86_h.jpg 1600w"
            style="margin: 2px; display: block; position: absolute; left: 379.7px; top: 508.8px;"
            title="DSC02492"
            width="375.7"
          />
          <img
            alt="DSC02489"
            height="250.4"
            sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
            src="https://live.staticflickr.com/65535/40673436913_c48ac9e6d5_b.jpg"
            srcset="https://live.staticflickr.com/65535/40673436913_c48ac9e6d5.jpg 500w,https://live.staticflickr.com/65535/40673436913_c48ac9e6d5_c.jpg 800w,https://live.staticflickr.com/65535/40673436913_c48ac9e6d5_b.jpg 1024w,https://live.staticflickr.com/65535/40673436913_dbb2320a41_h.jpg 1600w"
            style="margin: 2px; display: block; position: absolute; left: 759.3px; top: 508.8px;"
            title="DSC02489"
            width="375.7"
          />
          <img
            alt="DSC02479"
            height="225"
            sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
            src="https://live.staticflickr.com/65535/40673437213_3480e88c8b_b.jpg"
            srcset="https://live.staticflickr.com/65535/40673437213_3480e88c8b.jpg 500w,https://live.staticflickr.com/65535/40673437213_3480e88c8b_c.jpg 800w,https://live.staticflickr.com/65535/40673437213_3480e88c8b_b.jpg 1024w,https://live.staticflickr.com/65535/40673437213_5bfff3ef95_h.jpg 1600w"
            style="margin: 2px; display: block; position: absolute; left: 379.7px; top: 710.5px;"
            title="DSC02479"
            width="375.7"
          />
          <img
            alt="DSC02461"
            height="250.4"
            sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
            src="https://live.staticflickr.com/65535/47587238482_5ac51de790_b.jpg"
            srcset="https://live.staticflickr.com/65535/47587238482_5ac51de790.jpg 500w,https://live.staticflickr.com/65535/47587238482_5ac51de790_c.jpg 800w,https://live.staticflickr.com/65535/47587238482_5ac51de790_b.jpg 1024w,https://live.staticflickr.com/65535/47587238482_b8b1478145_h.jpg 1600w"
            style="margin: 2px; display: block; position: absolute; left: 0px; top: 763.2px;"
            title="DSC02461"
            width="375.7"
          />
          <img
            alt="DSC02460"
            height="250.4"
            sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
            src="https://live.staticflickr.com/65535/32697498227_a3bc50aa6c_b.jpg"
            srcset="https://live.staticflickr.com/65535/32697498227_a3bc50aa6c.jpg 500w,https://live.staticflickr.com/65535/32697498227_a3bc50aa6c_c.jpg 800w,https://live.staticflickr.com/65535/32697498227_a3bc50aa6c_b.jpg 1024w,https://live.staticflickr.com/65535/32697498227_e5a478e65c_h.jpg 1600w"
            style="margin: 2px; display: block; position: absolute; left: 759.3px; top: 763.2px;"
            title="DSC02460"
            width="375.7"
          />
          <img
            alt="DSC02456"
            height="255.6"
            sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
            src="https://live.staticflickr.com/65535/40673437263_b03be98bae_b.jpg"
            srcset="https://live.staticflickr.com/65535/40673437263_b03be98bae.jpg 500w,https://live.staticflickr.com/65535/40673437263_b03be98bae_c.jpg 800w,https://live.staticflickr.com/65535/40673437263_b03be98bae_b.jpg 1024w,https://live.staticflickr.com/65535/40673437263_bebc91464c_h.jpg 1600w"
            style="margin: 2px; display: block; position: absolute; left: 379.7px; top: 939.5px;"
            title="DSC02456"
            width="375.7"
          />
          <img
            alt="DSC02453"
            height="250.4"
            sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
            src="https://live.staticflickr.com/65535/32697497717_1fb25b32be_b.jpg"
            srcset="https://live.staticflickr.com/65535/32697497717_1fb25b32be.jpg 500w,https://live.staticflickr.com/65535/32697497717_1fb25b32be_c.jpg 800w,https://live.staticflickr.com/65535/32697497717_1fb25b32be_b.jpg 1024w,https://live.staticflickr.com/65535/32697497717_4bf3235964_h.jpg 1600w"
            style="margin: 2px; display: block; position: absolute; left: 0px; top: 1017.6px;"
            title="DSC02453"
            width="375.7"
          />
          <img
            alt="DSC02441"
            height="250.4"
            sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
            src="https://live.staticflickr.com/65535/47587238552_cfce31cc10_b.jpg"
            srcset="https://live.staticflickr.com/65535/47587238552_cfce31cc10.jpg 500w,https://live.staticflickr.com/65535/47587238552_cfce31cc10_c.jpg 800w,https://live.staticflickr.com/65535/47587238552_cfce31cc10_b.jpg 1024w,https://live.staticflickr.com/65535/47587238552_44cd3655ea_h.jpg 1600w"
            style="margin: 2px; display: block; position: absolute; left: 759.3px; top: 1017.6px;"
            title="DSC02441"
            width="375.7"
          />
          <img
            alt="DSC02439"
            height="250.4"
            sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
            src="https://live.staticflickr.com/65535/47640016631_cfffe28dd0_b.jpg"
            srcset="https://live.staticflickr.com/65535/47640016631_cfffe28dd0.jpg 500w,https://live.staticflickr.com/65535/47640016631_cfffe28dd0_c.jpg 800w,https://live.staticflickr.com/65535/47640016631_cfffe28dd0_b.jpg 1024w,https://live.staticflickr.com/65535/47640016631_ec4ec9f250_h.jpg 1600w"
            style="margin: 2px; display: block; position: absolute; left: 379.7px; top: 1199.1px;"
            title="DSC02439"
            width="375.7"
          />
          <img
            alt="DSC02436"
            height="250.4"
            sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
            src="https://live.staticflickr.com/65535/47640017061_d6f554a175_b.jpg"
            srcset="https://live.staticflickr.com/65535/47640017061_d6f554a175.jpg 500w,https://live.staticflickr.com/65535/47640017061_d6f554a175_c.jpg 800w,https://live.staticflickr.com/65535/47640017061_d6f554a175_b.jpg 1024w,https://live.staticflickr.com/65535/47640017061_cf1fd70288_h.jpg 1600w"
            style="margin: 2px; display: block; position: absolute; left: 0px; top: 1272px;"
            title="DSC02436"
            width="375.7"
          />
          <img
            alt="DSC02435"
            height="166.2"
            sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
            src="https://live.staticflickr.com/65535/32697497787_ebd9165050_b.jpg"
            srcset="https://live.staticflickr.com/65535/32697497787_ebd9165050.jpg 500w,https://live.staticflickr.com/65535/32697497787_ebd9165050_c.jpg 800w,https://live.staticflickr.com/65535/32697497787_ebd9165050_b.jpg 1024w,https://live.staticflickr.com/65535/32697497787_e3c6cb4bc5_h.jpg 1600w"
            style="margin: 2px; display: block; position: absolute; left: 759.3px; top: 1272px;"
            title="DSC02435"
            width="375.7"
          />
          <img
            alt="DSC02428"
            height="560.9"
            sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
            src="https://live.staticflickr.com/65535/40673437303_ec8b5a7246_b.jpg"
            srcset="https://live.staticflickr.com/65535/40673437303_ec8b5a7246.jpg 333w,https://live.staticflickr.com/65535/40673437303_ec8b5a7246_c.jpg 533w,https://live.staticflickr.com/65535/40673437303_ec8b5a7246_b.jpg 682w,https://live.staticflickr.com/65535/40673437303_c88d6174a2_h.jpg 1065w"
            style="margin: 2px; display: block; position: absolute; left: 759.3px; top: 1442.2px;"
            title="DSC02428"
            width="375.7"
          />
          <img
            alt="DSC02427"
            height="250.4"
            sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
            src="https://live.staticflickr.com/65535/46724351385_a6ff50ddda_b.jpg"
            srcset="https://live.staticflickr.com/65535/46724351385_a6ff50ddda.jpg 500w,https://live.staticflickr.com/65535/46724351385_a6ff50ddda_c.jpg 800w,https://live.staticflickr.com/65535/46724351385_a6ff50ddda_b.jpg 1024w,https://live.staticflickr.com/65535/46724351385_3a29cde51c_h.jpg 1600w"
            style="margin: 2px; display: block; position: absolute; left: 379.7px; top: 1453.5px;"
            title="DSC02427"
            width="375.7"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="react-photo-gallery--gallery"
    >
      <div
        style="position: relative; height: 2007.1px;"
      >
        <img
          alt="DSC02576"
          height="250.4"
          sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
          src="https://live.staticflickr.com/65535/40673436833_5fc1ab52fe_b.jpg"
          srcset="https://live.staticflickr.com/65535/40673436833_5fc1ab52fe.jpg 500w,https://live.staticflickr.com/65535/40673436833_5fc1ab52fe_c.jpg 800w,https://live.staticflickr.com/65535/40673436833_5fc1ab52fe_b.jpg 1024w,https://live.staticflickr.com/65535/40673436833_6f245db6aa_h.jpg 1600w"
          style="margin: 2px; display: block; position: absolute; left: 0px; top: 0px;"
          title="DSC02576"
          width="375.7"
        />
        <img
          alt="DSC02565"
          height="250.4"
          sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
          src="https://live.staticflickr.com/65535/47587238132_4683fdabb2_b.jpg"
          srcset="https://live.staticflickr.com/65535/47587238132_4683fdabb2.jpg 500w,https://live.staticflickr.com/65535/47587238132_4683fdabb2_c.jpg 800w,https://live.staticflickr.com/65535/47587238132_4683fdabb2_b.jpg 1024w,https://live.staticflickr.com/65535/47587238132_54f98ac5e9_h.jpg 1600w"
          style="margin: 2px; display: block; position: absolute; left: 379.7px; top: 0px;"
          title="DSC02565"
          width="375.7"
        />
        <img
          alt="DSC02548"
          height="250.4"
          sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
          src="https://live.staticflickr.com/65535/32697497947_7d6dfaf5ec_b.jpg"
          srcset="https://live.staticflickr.com/65535/32697497947_7d6dfaf5ec.jpg 500w,https://live.staticflickr.com/65535/32697497947_7d6dfaf5ec_c.jpg 800w,https://live.staticflickr.com/65535/32697497947_7d6dfaf5ec_b.jpg 1024w,https://live.staticflickr.com/65535/32697497947_81c6d9139e_h.jpg 1600w"
          style="margin: 2px; display: block; position: absolute; left: 759.3px; top: 0px;"
          title="DSC02548"
          width="375.7"
        />
        <img
          alt="DSC02540"
          height="250.4"
          sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
          src="https://live.staticflickr.com/65535/32697497647_31cd88c0b5_b.jpg"
          srcset="https://live.staticflickr.com/65535/32697497647_31cd88c0b5.jpg 500w,https://live.staticflickr.com/65535/32697497647_31cd88c0b5_c.jpg 800w,https://live.staticflickr.com/65535/32697497647_31cd88c0b5_b.jpg 1024w,https://live.staticflickr.com/65535/32697497647_7a36437b05_h.jpg 1600w"
          style="margin: 2px; display: block; position: absolute; left: 0px; top: 254.4px;"
          title="DSC02540"
          width="375.7"
        />
        <img
          alt="DSC02535"
          height="250.4"
          sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
          src="https://live.staticflickr.com/65535/47587238242_6776b8a582_b.jpg"
          srcset="https://live.staticflickr.com/65535/47587238242_6776b8a582.jpg 500w,https://live.staticflickr.com/65535/47587238242_6776b8a582_c.jpg 800w,https://live.staticflickr.com/65535/47587238242_6776b8a582_b.jpg 1024w,https://live.staticflickr.com/65535/47587238242_feab7e0aa0_h.jpg 1600w"
          style="margin: 2px; display: block; position: absolute; left: 379.7px; top: 254.4px;"
          title="DSC02535"
          width="375.7"
        />
        <img
          alt="DSC02520"
          height="250.4"
          sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
          src="https://live.staticflickr.com/65535/40673437163_703ffe416d_b.jpg"
          srcset="https://live.staticflickr.com/65535/40673437163_703ffe416d.jpg 500w,https://live.staticflickr.com/65535/40673437163_703ffe416d_c.jpg 800w,https://live.staticflickr.com/65535/40673437163_703ffe416d_b.jpg 1024w,https://live.staticflickr.com/65535/40673437163_7f4a4d66f2_h.jpg 1600w"
          style="margin: 2px; display: block; position: absolute; left: 759.3px; top: 254.4px;"
          title="DSC02520"
          width="375.7"
        />
        <img
          alt="DSC02494"
          height="250.4"
          sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
          src="https://live.staticflickr.com/65535/47587238362_5d355937cb_b.jpg"
          srcset="https://live.staticflickr.com/65535/47587238362_5d355937cb.jpg 500w,https://live.staticflickr.com/65535/47587238362_5d355937cb_c.jpg 800w,https://live.staticflickr.com/65535/47587238362_5d355937cb_b.jpg 1024w,https://live.staticflickr.com/65535/47587238362_65fe216f4b_h.jpg 1600w"
          style="margin: 2px; display: block; position: absolute; left: 0px; top: 508.8px;"
          title="DSC02494"
          width="375.7"
        />
        <img
          alt="DSC02492"
          height="197.7"
          sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
          src="https://live.staticflickr.com/65535/32697498187_7a74849c76_b.jpg"
          srcset="https://live.staticflickr.com/65535/32697498187_7a74849c76.jpg 500w,https://live.staticflickr.com/65535/32697498187_7a74849c76_c.jpg 800w,https://live.staticflickr.com/65535/32697498187_7a74849c76_b.jpg 1024w,https://live.staticflickr.com/65535/32697498187_9d07b79e86_h.jpg 1600w"
          style="margin: 2px; display: block; position: absolute; left: 379.7px; top: 508.8px;"
          title="DSC02492"
          width="375.7"
        />
        <img
          alt="DSC02489"
          height="250.4"
          sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
          src="https://live.staticflickr.com/65535/40673436913_c48ac9e6d5_b.jpg"
          srcset="https://live.staticflickr.com/65535/40673436913_c48ac9e6d5.jpg 500w,https://live.staticflickr.com/65535/40673436913_c48ac9e6d5_c.jpg 800w,https://live.staticflickr.com/65535/40673436913_c48ac9e6d5_b.jpg 1024w,https://live.staticflickr.com/65535/40673436913_dbb2320a41_h.jpg 1600w"
          style="margin: 2px; display: block; position: absolute; left: 759.3px; top: 508.8px;"
          title="DSC02489"
          width="375.7"
        />
        <img
          alt="DSC02479"
          height="225"
          sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
          src="https://live.staticflickr.com/65535/40673437213_3480e88c8b_b.jpg"
          srcset="https://live.staticflickr.com/65535/40673437213_3480e88c8b.jpg 500w,https://live.staticflickr.com/65535/40673437213_3480e88c8b_c.jpg 800w,https://live.staticflickr.com/65535/40673437213_3480e88c8b_b.jpg 1024w,https://live.staticflickr.com/65535/40673437213_5bfff3ef95_h.jpg 1600w"
          style="margin: 2px; display: block; position: absolute; left: 379.7px; top: 710.5px;"
          title="DSC02479"
          width="375.7"
        />
        <img
          alt="DSC02461"
          height="250.4"
          sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
          src="https://live.staticflickr.com/65535/47587238482_5ac51de790_b.jpg"
          srcset="https://live.staticflickr.com/65535/47587238482_5ac51de790.jpg 500w,https://live.staticflickr.com/65535/47587238482_5ac51de790_c.jpg 800w,https://live.staticflickr.com/65535/47587238482_5ac51de790_b.jpg 1024w,https://live.staticflickr.com/65535/47587238482_b8b1478145_h.jpg 1600w"
          style="margin: 2px; display: block; position: absolute; left: 0px; top: 763.2px;"
          title="DSC02461"
          width="375.7"
        />
        <img
          alt="DSC02460"
          height="250.4"
          sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
          src="https://live.staticflickr.com/65535/32697498227_a3bc50aa6c_b.jpg"
          srcset="https://live.staticflickr.com/65535/32697498227_a3bc50aa6c.jpg 500w,https://live.staticflickr.com/65535/32697498227_a3bc50aa6c_c.jpg 800w,https://live.staticflickr.com/65535/32697498227_a3bc50aa6c_b.jpg 1024w,https://live.staticflickr.com/65535/32697498227_e5a478e65c_h.jpg 1600w"
          style="margin: 2px; display: block; position: absolute; left: 759.3px; top: 763.2px;"
          title="DSC02460"
          width="375.7"
        />
        <img
          alt="DSC02456"
          height="255.6"
          sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
          src="https://live.staticflickr.com/65535/40673437263_b03be98bae_b.jpg"
          srcset="https://live.staticflickr.com/65535/40673437263_b03be98bae.jpg 500w,https://live.staticflickr.com/65535/40673437263_b03be98bae_c.jpg 800w,https://live.staticflickr.com/65535/40673437263_b03be98bae_b.jpg 1024w,https://live.staticflickr.com/65535/40673437263_bebc91464c_h.jpg 1600w"
          style="margin: 2px; display: block; position: absolute; left: 379.7px; top: 939.5px;"
          title="DSC02456"
          width="375.7"
        />
        <img
          alt="DSC02453"
          height="250.4"
          sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
          src="https://live.staticflickr.com/65535/32697497717_1fb25b32be_b.jpg"
          srcset="https://live.staticflickr.com/65535/32697497717_1fb25b32be.jpg 500w,https://live.staticflickr.com/65535/32697497717_1fb25b32be_c.jpg 800w,https://live.staticflickr.com/65535/32697497717_1fb25b32be_b.jpg 1024w,https://live.staticflickr.com/65535/32697497717_4bf3235964_h.jpg 1600w"
          style="margin: 2px; display: block; position: absolute; left: 0px; top: 1017.6px;"
          title="DSC02453"
          width="375.7"
        />
        <img
          alt="DSC02441"
          height="250.4"
          sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
          src="https://live.staticflickr.com/65535/47587238552_cfce31cc10_b.jpg"
          srcset="https://live.staticflickr.com/65535/47587238552_cfce31cc10.jpg 500w,https://live.staticflickr.com/65535/47587238552_cfce31cc10_c.jpg 800w,https://live.staticflickr.com/65535/47587238552_cfce31cc10_b.jpg 1024w,https://live.staticflickr.com/65535/47587238552_44cd3655ea_h.jpg 1600w"
          style="margin: 2px; display: block; position: absolute; left: 759.3px; top: 1017.6px;"
          title="DSC02441"
          width="375.7"
        />
        <img
          alt="DSC02439"
          height="250.4"
          sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
          src="https://live.staticflickr.com/65535/47640016631_cfffe28dd0_b.jpg"
          srcset="https://live.staticflickr.com/65535/47640016631_cfffe28dd0.jpg 500w,https://live.staticflickr.com/65535/47640016631_cfffe28dd0_c.jpg 800w,https://live.staticflickr.com/65535/47640016631_cfffe28dd0_b.jpg 1024w,https://live.staticflickr.com/65535/47640016631_ec4ec9f250_h.jpg 1600w"
          style="margin: 2px; display: block; position: absolute; left: 379.7px; top: 1199.1px;"
          title="DSC02439"
          width="375.7"
        />
        <img
          alt="DSC02436"
          height="250.4"
          sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
          src="https://live.staticflickr.com/65535/47640017061_d6f554a175_b.jpg"
          srcset="https://live.staticflickr.com/65535/47640017061_d6f554a175.jpg 500w,https://live.staticflickr.com/65535/47640017061_d6f554a175_c.jpg 800w,https://live.staticflickr.com/65535/47640017061_d6f554a175_b.jpg 1024w,https://live.staticflickr.com/65535/47640017061_cf1fd70288_h.jpg 1600w"
          style="margin: 2px; display: block; position: absolute; left: 0px; top: 1272px;"
          title="DSC02436"
          width="375.7"
        />
        <img
          alt="DSC02435"
          height="166.2"
          sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
          src="https://live.staticflickr.com/65535/32697497787_ebd9165050_b.jpg"
          srcset="https://live.staticflickr.com/65535/32697497787_ebd9165050.jpg 500w,https://live.staticflickr.com/65535/32697497787_ebd9165050_c.jpg 800w,https://live.staticflickr.com/65535/32697497787_ebd9165050_b.jpg 1024w,https://live.staticflickr.com/65535/32697497787_e3c6cb4bc5_h.jpg 1600w"
          style="margin: 2px; display: block; position: absolute; left: 759.3px; top: 1272px;"
          title="DSC02435"
          width="375.7"
        />
        <img
          alt="DSC02428"
          height="560.9"
          sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
          src="https://live.staticflickr.com/65535/40673437303_ec8b5a7246_b.jpg"
          srcset="https://live.staticflickr.com/65535/40673437303_ec8b5a7246.jpg 333w,https://live.staticflickr.com/65535/40673437303_ec8b5a7246_c.jpg 533w,https://live.staticflickr.com/65535/40673437303_ec8b5a7246_b.jpg 682w,https://live.staticflickr.com/65535/40673437303_c88d6174a2_h.jpg 1065w"
          style="margin: 2px; display: block; position: absolute; left: 759.3px; top: 1442.2px;"
          title="DSC02428"
          width="375.7"
        />
        <img
          alt="DSC02427"
          height="250.4"
          sizes="(min-width: 480px) 50vw, (min-width: 1024px) 33.3vw, 100vw"
          src="https://live.staticflickr.com/65535/46724351385_a6ff50ddda_b.jpg"
          srcset="https://live.staticflickr.com/65535/46724351385_a6ff50ddda.jpg 500w,https://live.staticflickr.com/65535/46724351385_a6ff50ddda_c.jpg 800w,https://live.staticflickr.com/65535/46724351385_a6ff50ddda_b.jpg 1024w,https://live.staticflickr.com/65535/46724351385_3a29cde51c_h.jpg 1600w"
          style="margin: 2px; display: block; position: absolute; left: 379.7px; top: 1453.5px;"
          title="DSC02427"
          width="375.7"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
